app/utils/geostore.py

```diff
@@ -8,29 +7,36 @@
 from app.crud.geostore import get_geostore_from_anywhere
 from app.errors import BadResponseError, InvalidResponseError, RecordNotFoundError
 from app.models.enum.geostore import GeostoreOrigin
-from app.models.pydantic.geostore import Geometry, Geostore
+from app.models.pydantic.geostore import Geostore, GeostoreCommon
 from app.utils import rw_api
 
 
 @alru_cache(maxsize=128)
-async def _get_gfw_geostore_geometry(geostore_id: UUID) -> Geometry:
+async def _get_gfw_geostore_geometry(geostore_id: UUID) -> GeostoreCommon:
     """Get GFW Geostore geometry."""
 
     try:
         geostore: Geostore = await get_geostore_from_anywhere(geostore_id)
-        geometry: Optional[Geometry] = geostore.gfw_geojson
+        geostore_common: GeostoreCommon = GeostoreCommon(
+            geostore_id=geostore.gfw_geostore_id,
+            geojson=geostore.gfw_geojson,
+            area__ha=geostore.gfw_area__ha,
+            bbox=geostore.gfw_bbox,
+        )
     except (KeyError, RecordNotFoundError) as ex:
         logger.exception(ex)
         raise BadResponseError("Cannot fetch geostore geometry")
 
-    if geometry is None:
+    if geostore.gfw_geojson is None:
```

Comment:
Hmm, is this right? Shouldn't the attribute name be just "geojson", so `if geostore.geojson is None:`?

Reply:
Oh, I guess either way is correct, because I have copy of both the GFW geostore and geostore_common